src/providers/ipa/ipa_common.c

```diff
@@ -1499,10 +1499,6 @@ ipa_create_trust_options(TALLOC_CTX *mem_ctx,
         }
     }
 
-    /* Set IPA KRB5 realm */
-    ret = dp_opt_set_string(ipa_options->id->basic,
-                            IPA_KRB5_REALM, subdom->realm);
-
```

Comment:
Hi,

yes, this is wrong because `id->basic` are the default LDAP/SDAP options and `IPA_KRB5_REALM` cannot be used here.

Funny side note, this is only place the return value is not checked and it turned out it is actually `EINVAL` here.

bye,
Sumit